demos/serverless/README.md

```diff
@@ -23,7 +23,7 @@ API Gateway deployment that runs the `meeting` demo.
 
 ```
 cd demos/serverless
-node ./deploy.js -r us-east-1 -b <my-bucket> -s <my-stack-name> -a meeting
+npm run deploy -r us-east-1 -b <my-bucket> -s <my-stack-name> -a meeting
```

Comment:
Make sure to change other files running `npm ./deploy.js`

https://github.com/aws/amazon-chime-sdk-js/blob/a8eac1cc2ac7fe66f3a72ef2c9aa04da761beb41/script/deploy-canary-demo#L11

https://github.com/aws/amazon-chime-sdk-js/blob/a8eac1cc2ac7fe66f3a72ef2c9aa04da761beb41/script/deploy-canary-demo#L18

https://github.com/aws/amazon-chime-sdk-js/blob/a8eac1cc2ac7fe66f3a72ef2c9aa04da761beb41/script/postpublish#L51

Reply:
Good catch. Updated.